routes/users-unit.test.js

```diff
@@ -0,0 +1,21 @@
+import app from "../app";
+import request from "supertest";
+
+import { getUsersByID } from "../helpers/users";
+
+import { jest } from "@jest/globals";
```

Comment:
This is the part I just could not get to work. It didn't like mocking the helper file. I think it's because it isn't a class.